Crawler.py

```diff
@@ -0,0 +1,28 @@
+import requests
+# This is needed to search in the html response specific elements
+from bs4 import BeautifulSoup
+
+# Creating Crawler Class
+class Crawler:
+    # Creating object that contains all websites
+    websites = {
+        'Dekmantel': 'https://www.dekmantelfestival.com/en/tickets/',
+        'Sonar': 'https://sonar.es/es/2021/tickets'
+    }
+    # Elements that need to be tracked
+    htmlElements = {
+        'Dekmantel': 'tickets',
+        'Sonar': 'main-container sm-full-width page_tickets lg-margin-top'
+        }
+    #Creating run call that crawles websites
+    def run(self):
+        for website, element in zip(self.websites,self.htmlElements):
```

Comment:
When you do `for website in self.websites` you're only reading the keys of the dictionary but not the values. If you want to read both name and website you can do `for event, website in self.websites.items()`

Since in this case you have two dictionaries to iterate it's maybe cleaner if you have a separate variable with a list of all even names. Then you could do:
`for event in events:`
     `eventWebsite = websites[event]`
     `eventElement = elements[event]`

Reply:
Not sure if I get this a 100%. So should I create a new list with events like event = [Dekmantel, Sonar,...] and then iterate over the two dictionaries?